diff -urN a/nmake.opt b/nmake.opt
--- a/nmake.opt	2019-10-28 12:56:15.000000000 +0300
+++ b/nmake.opt	2022-01-24 12:54:48.414017600 +0300
@@ -280,9 +280,9 @@
 !ENDIF
 
 # This will enable 12bit libjpeg - use only with internal jpeg builds.
-!IFNDEF JPEG12_SUPPORTED
-JPEG12_SUPPORTED = 1
-!ENDIF
+#!IFNDEF JPEG12_SUPPORTED
+#JPEG12_SUPPORTED = 1
+#!ENDIF
 
 #if using an external jpeg library uncomment the following lines
 #JPEG_EXTERNAL_LIB = 1
